cmd/internal/server/levels.go

```diff
@@ -14,7 +14,7 @@ var levelMap = map[string]log.Level{
 	"trace": log.TraceLevel,
 }
 
-func ParseLevel(s string) (log.Level, error) {
+func parseLevel(s string) (log.Level, error) {
```

Comment:
Maybe bake this in with the Server object's code? It's small enough.

Reply:
Done. I wish there were a more concise way to construct the error, but not finding good options.